src/js/src/components/inlinePlayer/InlinePlayer.vue

```diff
@@ -80,18 +80,16 @@ export default {
           controls: true,
           width: 400, 
           height: this.result.type === 'Video' ? 200 : 30,
-				  sources: [
-					{ 
-						src: this.result.url_norm,
-					}
-				]
+				  sources: this.getSource() 
         }
         return playerConf    
       },
-    
-      getFileExtension() {
-        return this.result.url_norm.match(/\.[0-9a-z]+$/i)
```

Comment:
my regex is not fantastic. Perhaps a little // explanation here about what goes on and why this is needed?